src/components/menu/menu.stories.mdx

```diff
@@ -53,210 +47,68 @@ import {
 ## Examples
 
 ### Default
-
 <Canvas>
-  <Story name="default" parameters={{ chromatic: { disableSnapshot: true } }}>
-    <Box mb={150}>
-      {["white", "light", "dark", "black"].map((menuType) => (
-        <div key={menuType}>
-          <Typography variant="h4" textTransform="capitalize" my={2}>
-            {menuType}
-          </Typography>
-          <Menu menuType={menuType}>
-            <MenuItem href="#">Menu Item One</MenuItem>
-            <MenuItem href="#">Menu Item Two</MenuItem>
-            <MenuItem submenu="Menu Item Three">
-              <MenuItem href="#">Item Submenu One</MenuItem>
-              <MenuItem href="#">Item Submenu Two</MenuItem>
-              <MenuDivider />
-              <MenuItem icon="settings" href="#">
-                Item Submenu Three
-              </MenuItem>
-              <MenuItem href="#">Item Submenu Four</MenuItem>
-            </MenuItem>
-            <MenuItem submenu="Menu Item Four" onClick={() => {}}>
-              <MenuItem onClick={() => {}}>Item Submenu One</MenuItem>
-              <MenuItem href="#">Item Submenu Two</MenuItem>
-            </MenuItem>
-          </Menu>
-        </div>
-      ))}
-    </Box>
-  </Story>
+  <Story
+    name="default"
+    parameters={{ chromatic: { disableSnapshot: true } }}
```

Comment:
We may want to move the parameters from this file to the `.stories.tsx` file, as they will be ignored here by storybook since they must be defined the same location as the stories themselves.